docs/runbooks/CDINoDefaultStorageClass.md

```diff
@@ -2,37 +2,39 @@
 
 ## Meaning
 
-This alert fires when there is no default (Kubernetes or KubeVirt) storage
-class, and a data volume is pending for one.
+This alert fires when there is no default (Kubernetes or virtualization) storage
+class, and a data volume is `Pending` for one.
 
-A default KubeVirt storage class has precedence over a default Kubernetes
+A default virtualization storage class has precedence over a default Kubernetes
 storage class for creating a VirtualMachine disk image.
 
 ## Impact
 
-If there is no default Kubernetes or KubeVirt storage class, a data volume that
-does not have a specified storage class remains in a "pending" state.
+If there is no default (Kubernetes or virtualization) storage class, a data
```

Comment:
Again, not sure if this needs to be parenthetical; if we need to specify it, we can just take the parentheses out: `If there is no default Kubernetes or virtualization storage class,`

Reply:
Maybe better neither … nor?

Reply:
Your suggestion to use neither/nor makes sense, but I wonder if that would be clear enough for translation. It can sound overly flowery, IMO. :) Here's an example:

`If there is neither a default Kubernetes storage class nor a default virtualization storage class,` 

Using "and" might eliminate ambiguity:

`If there is no default Kubernetes storage class and no default virtualization storage class,`